symbulate/plot.py

```diff
@@ -40,4 +41,117 @@ def plot(*args, **kwargs):
     except:
         plt.plot(*args, **kwargs)
         
+def is_discrete(heights):
+    return sum([(i > 1) for i in heights]) > .8 * len(heights)
+
+def count_var(x):
+    counts = {}
+    for val in x:
+        if val in counts:
+            counts[val] += 1
+        else:
+            counts[val] = 1
+    return counts
+    
+def compute_density(values):
+	density = gaussian_kde(values)
+	density.covariance_factor = lambda: 0.25
+	density._compute_covariance()
+	return density
+    
+def add_colorbar(fig, type, mappable, label):
+    if 'marginal' not in type:
```

Comment:
It looks like this is only getting called if there's a colorbar. Is this the desired behavior?

Reply:
Yeah that's the intent here. We want the colorbar to on the left of the plot and the ideal way to do this is to give it its own axis. The reason for the if statement here is to adjust the size of the colorbar when there are marginal plots.